packages/cln_plugin/lib/src/cln_plugin_base.dart

```diff
@@ -26,9 +26,9 @@ class Plugin implements ICLNPlugin {
     print("The plugin is started.");
```

Comment:
```suggestion
    // print("The plugin is started.");
```

This need to be removed, because the string `"The plugin is started."` is sent directly to core lightning, and core lightning kill the plugin because it is an unexpected string.

We need to log on file, or with core lightning, but with core lightning is difficult.

The solution that I can propose is to open a file and print the stuff on the file, so you can see what happens.

A small doc of this is available here https://docs.flutter.dev/cookbook/persistence/reading-writing-files if you want help I can open a PR with the logging implementation, should be easy to implement for me

Reply:
I think the final goal would be to log into core lightning, so maybe in a few days we can start working on implementing the core lightning logger. Does this sound fine?